packages/css/src/components/breakout/breakout.scss

```diff
@@ -3,105 +3,105 @@
  * Copyright Gemeente Amsterdam
  */
 
-@import "../../common/breakpoint";
-@import "../grid/grid";
-@import "../grid/mixins";
+@use "../../common/breakpoint";
+@use "../grid/grid";
+@use "../grid/mixins";
 
 $ams-breakout-row-span-max: 4;
 
 .ams-breakout {
-  @include ams-grid;
+  @include mixins.ams-grid;
```

Comment:
There is a significant chance of conflicts here if multiple components expose a file called `mixins`.

Reply:
It should only import grid as *, only column count should be moved and included as a sass var like so `grid.$column-count`. Is that an option?

Reply:
We could also export the mixins from `grid.scss`. Let’s discuss tomorrow.